guides/common/modules/proc_salt-using-content-management.adoc

```diff
@@ -0,0 +1,23 @@
+[id="salt_guide_using_content_management_{context}"]
+= Using {Project}s Content Management
+
+This section provides best practice recommendations on how to provide managed hosts with the required Salt Minion client software using {Project}.
+
+.Procedure
+. Create and synchronize the required products and repositories.
+.. Create a product called `Salt`.
```

Comment:
You could link to content management docs on how to create a product?

Reply:
Added multiple links.